autoscheduler/frontend/src/utils/sortMeetingFunction.tsx

```diff
@@ -0,0 +1,8 @@
+import Meeting from '../types/Meeting';
+
+// sorts meeetings based on meeting type
```

Comment:
```suggestion
/** Sorts meetings based on meeting type */
```
A couple things:
- `meetings` is spelled wrong
- If you wrap a comment with `/** */`, it will be a docstring (meaning that when it autocompletes, you'll be able to see this text as a reminder of what the function does